File: VB/SpreadWinDemo/rowcolcell/insertdialog.vb
Public Class insertdialog

    Public Sub New()

        InitializeComponent()

        ' ワークブックの設定
        InitWorkbook(FpSpread1.AsWorkbook())
    End Sub

    Private Sub InitWorkbook(ByVal workbook As GrapeCity.Spreadsheet.IWorkbook)
        ' フォントの設定
        Dim normalStyle As GrapeCity.Spreadsheet.IStyle = workbook.Styles(GrapeCity.Spreadsheet.BuiltInStyle.Normal)
        normalStyle.Font.Name = "メイリオ"
        normalStyle.Font.Size = 9

        FpSpread1.Features.ExcelCompatibleKeyboardShortcuts = True
        FpSpread1.Features.RichClipboard = True
        Dim sheet1 As GrapeCity.Spreadsheet.IWorksheet = workbook.ActiveSheet
        For i As Integer = 0 To 5 - 1
            For j As Integer = 0 To 5 - 1
                sheet1.Cells(i, j).Value = i + j
            Next
        Next
    End Sub
End Class

File: CS/SpreadWinDemoCS/rowcolcell/insertdialog.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace SpreadWinDemo.rowcolcell
{
    public partial class insertdialog : SpreadWinDemo.DemoBase
    {
        public insertdialog()
        {
            InitializeComponent();

            // ワークブックの設定
            InitWorkbook(fpSpread1.AsWorkbook());
        }

        private void InitWorkbook(GrapeCity.Spreadsheet.IWorkbook workbook)
        {
            // フォントの設定
            GrapeCity.Spreadsheet.IStyle normalStyle = workbook.Styles[GrapeCity.Spreadsheet.BuiltInStyle.Normal];
            normalStyle.Font.Name = "メイリオ";
            normalStyle.Font.Size = 9;

            fpSpread1.Features.ExcelCompatibleKeyboardShortcuts = true;
            fpSpread1.Features.RichClipboard = true;
            GrapeCity.Spreadsheet.IWorksheet sheet1 = workbook.ActiveSheet;
            for (int i = 0; i < 5; i++)
            {
                for (int j = 0; j < 5; j++)
                {
                    sheet1.Cells[i, j].Value = i + j;
                }
            }
        }
    }
}
